unidist/core/backends/dask/backend.py

```diff
@@ -151,6 +153,26 @@ def num_cpus():
         """
         return len(get_client().ncores())
 
+    @staticmethod
+    def cluster_resources():
+        """
+        Get resources of Dask cluster.
+
+        Returns
+        -------
+        dict
+            Dictionary with cluster nodes info in the form '{node_ip0: {CPU: x0},
+            node_ip1: {CPU: x1}, ..}'.
+        """
+        client = get_client()
+        cluster_resources = defaultdict(lambda: {"CPU": 0})
```

Comment:
The purpose is to have start value "0", because increment is used below. MPI implementation uses simple `defaultdict(dict)` because only assignments by key is happens, no increment.